stopes/pipelines/asr_bleu/compute_asr_bleu.py

```diff
@@ -0,0 +1,65 @@
+import asyncio
+import logging
+
+from stopes.core import utils
+from stopes.pipelines.asr_bleu.configs import AsrBleuConfig
+from stopes.pipelines.asr_bleu.utils import retrieve_asr_config
+from stopes.pipelines.asr_bleu.transcribe_audio import transcribe_audio
+from stopes.pipelines.asr_bleu.retrieve_data import retrieve_data
+
+import hydra
+import sacrebleu
+
+logger = logging.getLogger("stopes.asr_bleu")
+
+class AsrBleu:
+    def __init__(self, config: AsrBleuConfig):
+        self.config = config
+        self.launcher = hydra.utils.instantiate(self.config.launcher)
+        utils.ensure_dir(self.config.output_dir)
```

Comment:
If I understand correctly, this script will print the BLEU score, but the transcriptions won't be saved anywhere. 
Does it mean that `output_dir` is going to be unused?